client/gandi.go

```diff
@@ -1,4 +1,4 @@
-package gandi
+package client
```

Comment:
Is it wise to have a package named "client"? It almost guarantees that people will rename it on import.  Most people will rename it to gandi or gandiclient, so why not use one of those?

Reply:
Moreover, why a "client" package when there is no "server" ?

Reply:
Given that there will never (?) be a server, I recommend the name "gandi".

Reply:
IMHO it should remain at the root of the repository. Why moving these files ?

Reply:
> Moreover, why a "client" package when there is no "server" ?

It's a client of the Gandi API. I'm fine with changing the name back, although the way I've structured this is that most of the time the client should be invisible, and you'd just do `LiveDNS.New()`. For the same reason I've moved it out of the root; the packages you actually want are the ones that interact with the parts of the API, not the thing that makes HTTP requests.

Reply:
Perfect reason why this package should not be called "client". Because it will not be used as a client.
Maybe something like "apiconnection" ?

Reply:
Oh btw, it should probably be under `internal/`. Thus, it may be called `client` after all, but as an internal package, that way we cannot import it thinking it is a client to be used by another library.

=> `[...]/go-gandi-livedns/internal/client`